<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>preferences</class>
 <widget class="QDialog" name="preferences">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>554</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="toolTipDuration">
   <number>0</number>
  </property>
  <widget class="QPushButton" name="generalButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="toolTipDuration">
    <number>0</number>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/home-general-icon-2.png</normaloff>:/images/home-general-icon-2.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>60</width>
     <height>60</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="editButton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>20</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/edit.png</normaloff>:/images/edit.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>60</width>
     <height>60</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="themeButton">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>20</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/themes.png</normaloff>:/images/themes.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>60</width>
     <height>60</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="cssButton">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>20</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/css.png</normaloff>:/images/css.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>60</width>
     <height>60</height>
    </size>
   </property>
  </widget>
  <widget class="QComboBox" name="t_part2">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>150</y>
     <width>104</width>
     <height>26</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Paper</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Paper+</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Tomorrow</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Tomorrow Blue</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Fresh Air</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>59</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>.Helvetica Neue DeskInterface</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>General</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>90</y>
     <width>59</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Edit</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>90</y>
     <width>59</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>themes</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>90</y>
     <width>59</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>CSS</string>
   </property>
  </widget>
  <widget class="QLabel" name="t_part1">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>150</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>use theme</string>
   </property>
  </widget>
  <widget class="QPushButton" name="t_part3">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>145</y>
     <width>50</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Edit</string>
   </property>
  </widget>
  <widget class="QLabel" name="e_part1">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>150</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Base Font</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="e_part2">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>150</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Helvitica</string>
   </property>
  </widget>
  <widget class="QPushButton" name="e_part3">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>150</y>
     <width>61</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>fonts</string>
   </property>
  </widget>
  <widget class="QLabel" name="e_part5">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>200</y>
     <width>261</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Make links clickable in Editor View</string>
   </property>
  </widget>
  <widget class="QLabel" name="e_part7">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>230</y>
     <width>261</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Insert 4 spaces instead of one tab</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="e_part9">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>260</y>
     <width>47</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QRadioButton" name="e_part4">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>200</y>
     <width>21</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QRadioButton" name="e_part6">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>230</y>
     <width>21</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="e_part8">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>260</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Line Spacing</string>
   </property>
  </widget>
  <widget class="QLabel" name="c_part1">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>150</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Use CSS</string>
   </property>
  </widget>
  <widget class="QComboBox" name="c_part2">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>150</y>
     <width>104</width>
     <height>26</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Github2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Github</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ClearnessDark</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="c_part3">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>150</y>
     <width>61</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Edit</string>
   </property>
  </widget>
  <widget class="QLabel" name="g_part1">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>150</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>All Document</string>
   </property>
  </widget>
  <widget class="QLabel" name="g_part2">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>150</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Play KeyPress Sound</string>
   </property>
  </widget>
  <widget class="QLabel" name="g_part3">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>180</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Enable Sync Scroll</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="g_part4">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>150</y>
     <width>21</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QRadioButton" name="g_part5">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>180</y>
     <width>21</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="g_part6">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>250</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Default File Extensions</string>
   </property>
  </widget>
  <widget class="QComboBox" name="g_part7">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>260</y>
     <width>104</width>
     <height>26</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>*.md</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>*.pdf</string>
    </property>
   </item>
  </widget>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
